test/test-script.js

```diff
@@ -387,44 +387,48 @@ describe('Player', function() {
   }); // end .equippedWith specs
 }); // end Player specs
 
+var zombie
```

Comment:
`var zombie;` should go inside `describe('Zombie'... `
just before `beforeEach`